Bunch of random quest fixes & additions

h3tR opened the pull request.

Markh3tr committed:
Some quest fixes and additions

Markh3tr committed:
Merge branch 'master-ceu' into quests

Markh3tr committed:
Dronepad quest + quest fixes

Markh3tr committed:
Lang fix

planetme commented on config/betterquesting/resources/supersymmetry/lang/en_us.lang:
```diff
@@ -2008,6 +2012,8 @@ susy.quest.db.1867050010.title=Better Kiln
 susy.quest.db.1867050010.desc=The §6Refractory Kiln§r is an upgraded version of the §7Stone Kiln§r, which has §a2.5x§r processing speed and a lower chance of failure.
 susy.quest.db.1902566957.title=Not Enough Fibers
 susy.quest.db.1902566957.desc=§6Drying Racks§r can convert §aPlant Fibers§r into §4Dried Plant Fibers§r. If the dried plant fibers obtained by breaking grass are insufficient for your progress, be sure to craft some.%n%nA §6Drying Rack§r operates more quickly in §edry§r biomes and at a slower rate in §bwet§r biomes. The weather may also influence its efficiency. Additionally, if the drying rack i ...
+susy.quest.db.1934888714.title=Commando gear
```
capitalize geat plus maybe add some description?

planetme commented on config/betterquesting/resources/supersymmetry/lang/en_us.lang:
```diff
@@ -2016,6 +2022,8 @@ susy.quest.db.1996974870.title=Transesterification Reaction
 susy.quest.db.1997511526.title=Natural Gas
 susy.quest.db.1997511526.desc=No Description
+susy.quest.db.1999167293.title=IR Transformers
+susy.quest.db.1999167293.desc=You can use Industrial Renewals, Ultra-HV transformers to economically transfer energy over long distances.
```
Industrial Renewal's Ultra-HV transformers*

planetme commented on config/betterquesting/resources/supersymmetry/lang/en_us.lang:
```diff
@@ -1938,6 +1940,8 @@ susy.quest.db.1319198702.title=Jet Wingpack
 susy.quest.db.1319198702.desc=The §3Jet Wingpack§r represents an advancement in §6Jetpack§r technology. By integrating a §eWingsuit System§r with four kerosene-powered §4Turbojets§r, this device enables users to achieve flight faster than ever before.%n%nTo begin gliding, simply press §6[Space]§r once you are off the ground. If the §4Turbojets§r are running, you can take off with a single press o ...
 susy.quest.db.1322263999.title=Air Separation Mark II
 susy.quest.db.1322263999.desc=§bCryogenic air separation§r is a technique to produce massive volumes of liquid and gaseous atmospheric elements for use in your base. It is also the only method to produce the stable noble gases heavier than §ehelium§r, which can also be produced from natural gas. This will prove useful for superconductor manufacturing and eventually fueling your rockets to head of ...
+susy.quest.db.1324712196.title=Drone pad
```
capitalize pad

planetme commented on config/betterquesting/resources/supersymmetry/lang/en_us.lang:
```diff
@@ -1938,6 +1940,8 @@ susy.quest.db.1319198702.title=Jet Wingpack
 susy.quest.db.1322263999.title=Air Separation Mark II
 susy.quest.db.1322263999.desc=§bCryogenic air separation§r is a technique to produce massive volumes of liquid and gaseous atmospheric elements for use in your base. It is also the only method to produce the stable noble gases heavier than §ehelium§r, which can also be produced from natural gas. This will prove useful for superconductor manufacturing and eventually fueling your rockets to head of ...
+susy.quest.db.1324712196.title=Drone pad
+susy.quest.db.1324712196.desc=With the drone pad you can automate a bunch of mob drops.
```
please include examples, i.e. wool, meat, etc.

planetme reviewed (changes requested):
also, could we get a screenshot of the new quests

Markh3tr committed:
fix typos + improved descriptions

Markh3tr committed:
update gitignore

h3tR committed:
Merge branch 'master-ceu' into quests

Markh3tr committed:
add creative hotbar to gitignore

h3tR commented:
I removed the drone quest from this PR in favour of the one already merged.
I've also added hotbar.nbt and shared object (so) files to the gitignore.

Markh3tr committed:
actually remove the old drone quest json and lang entry

Markh3tr committed:
How does this even happen

h3tR committed:
Merge branch 'SymmetricDevs:master-ceu' into quests

h3tR committed:
Add Kaminsky Catalyst Quest

h3tR commented:
I've added a new quest for the kaminsky catalyst under the zirconium quest
![image](https://github.com/user-attachments/assets/b939b843-37e4-403a-8f2f-5630bdfe25f0)

bruberu committed:
fix: formatting

bruberu committed:
fix: remove index.toml

bruberu commented:
Given that planetme is currently unavailable, I checked this, and it appears fine. Do note that you should try to run the questbook autoformatter when you make changes, since otherwise it leaves a large number of changes throughout all of the files.
